File: January 2022/Day 19/Que3.py
# Que link: https://leetcode.com/problems/ransom-note/

class Solution:
    def canConstruct(self, ransomNote: str, magazine: str) -> bool:
        ransomNote = [i for i in ransomNote]
        magazine = [i for i in magazine]

        for i in ransomNote:
            if i in magazine:
                magazine.remove(i)
            else:
                return False
        return True

File: January 2022/Day 12/Que3.java
// Que LINK: https://leetcode.com/problems/palindrome-number/

public class Solution {
    public boolean isPalindrome(int x) {
        if(x < 0) return false;
        int y = x;
        int ans = 0;
        while(y != 0) {
            ans = ans * 10 + y % 10;
            y /= 10;
        }
        return x == ans;
    }
}